src/js/edu-benefits/pages/applicantInformation.js

```diff
@@ -0,0 +1,59 @@
+import _ from 'lodash/fp';
+
+import * as currentOrPastDate from '../../common/schemaform/definitions/currentOrPastDate';
+import * as fullName from '../../common/schemaform/definitions/fullName';
+import * as ssn from '../../common/schemaform/definitions/ssn';
+
+import { relationshipLabels, genderLabels } from '../utils/helpers';
+
+const defaultProps = [
+  'relativeFullName',
+  'relativeSocialSecurityNumber',
+  'relativeDateOfBirth',
+  'gender',
+  'relationship'
+];
+
+export default function applicantInformation(
+  schema,
+  fields = defaultProps,
+  required = ['relativeFullName']) {
+  const possibleProperties = {
+    relativeFullName: fullName.schema,
+    relativeSocialSecurityNumber: ssn.schema,
+    relativeDateOfBirth: currentOrPastDate.schema,
```

Comment:
Not the `currentOrPastDate`, right? I was thinking about doing that for the `fullName` and `ssn`, but I figured in those files, they're just being pulled from `definitions.json` which is what the schemas are constructed from, but they may also have additional things (validations) tacked on them.

Also, I think `fullName` is defined in the `properties` of one of the schemas and the `definitions` of another, so I thought it would be inconsistent to have to include the definition in the config only sometimes when I'm doing the exact same thing on two different schemas.

Should I still go ahead and pull these from the `schema` param?